### PR TITLE
Update router to accept optionnal AliasId & scheme

### PR DESCRIPTION
**Pull request by ngilain**

### CODE REVIEW

File: FrontBundle/Manager/SitemapManager.php [hunk 1]
```diff
@@ -99,7 +99,13 @@ protected function generateNodeInfos(ReadNodeInterface $node, ReadSiteInterface
         if (is_null($node->getRole())) {
             try {
                 $nodeInfos = array(
-                    'loc' => $this->router->generate($node->getId(), array('aliasId' => $site->getMainAliasId())),
+                    'loc' => $this->router->generate(
+                        $node->getId(),
+                        array('required'=> array(
+                            'aliasId' => $site->getMainAliasId(),
+                        )),
+                        true
```
Comment (nicolasThal) [inline FrontBundle/Manager/SitemapManager.php]: Wouldn't you put the constant UrlGeneratorInterface::ABSOLUTE_PATH

Comment (ngilain) [inline FrontBundle/Manager/SitemapManager.php]: Good idea :) 
ABSOLUTE_URL is the right constant

**Comment by ngilain** [comment]
@nicolasThal : update done

**Comment by nicolasThal** [comment]
:+1: even if the build is failling

**Pull request merged**